### DISCUSSION TITLE
me and I

### DISCUSSION THREAD

**Discussion by u-an-i** [albar965/littlenavmap#669] [Show and tell]
u-an-i == u'n i

u'n i == u and i

u == you

https://www.youtube.com/watch?v=II4YQNThLQU&list=PL6-FaqQ8GMrxDeMtDG7rXuySduXqOPFYY

https://www.youtube.com/watch?v=fbdwL6-3qs4&list=PL6-FaqQ8GMrx5pdkx79buD3AvhzlEPgZ9

https://www.youtube.com/watch?v=D-3Hd1STlko&list=PL6-FaqQ8GMrwl4BtEEQts4IZPgpXv6VCQ

**Comment by albar965** [comment]
:slightly_smiling_face:
